package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "build": "npm run clean && npm run check-style && node_modules/webpack/bin/webpack.js",
     "build-style": "cd node_modules/cloudgov-style && npm install && npm run build",
-    "check-style": "/bin/bash -c '[[ $(npm ls cloudgov-style) =~ \"github.com\" ]] && npm run build-style || exit 0'",
+    "check-style": "/bin/bash -c '[[ $(npm ls cloudgov-style) =~ \"git://github.com\" ]] && npm run build-style || exit 0'",
```

Comment:
So currently, if I run `check-style` locally in any circumstance, it will trigger because some local path is `github.com/18F/cg-dashboard`. This fixes that.